footer.php

```diff
@@ -1,8 +1,50 @@
-  <footer>
-    <p>This is my footer.</p>
+</body>
+
+<footer>
+
+<div class="container-fluid text-center">
+  <div class="row align-items-center bg-secondary" style="footer">
+    <div class="col-sm-2">
+      <img src="<?php echo get_template_directory_uri() . '/assets/images/ClimateHubLogo.gif'; ?>" class="img-fluid border-0 chlogo"/>
+    </div>
+    <div class="col-sm-2">
+    <?php get_template_part( 'template-parts/footer_customfield' )?>
+    Creating a vibrant community around bold climate action at UBC.
+    </div>
+    <div class="col-sm-4 text-center mt-2 mb-2">
+
+      <h6 class="font-weight-bold ">FOLLOW US</h6>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <a href="https://www.facebook.com/UBCClimateHub/" class="fa fa-facebook"></a>
+        <a href="#" class="fa fa-twitter icon"></a>
+        <a href="#" class="fa fa-instagram"></a>

```

Comment:
Is there a reason that these 2 hrefs don't have a link?

Reply:
I wanted to leave it blank so that we could add it once they get a twitter/instagram (a placeholder of sort). I just sent a quick message to Liam to see if they would ever get twitter/instagram - anything besides FB

Reply:
Okay that's fine.